package.json

```diff
@@ -24,9 +24,20 @@
   },
   "homepage": "https://github.com/anewman15/restaurant-page#readme",
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
     "css-loader": "^5.0.1",
+    "eslint": "^7.14.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-plugin-import": "^2.22.1",
+    "sass": "^1.29.0",
+    "sass-loader": "^10.1.0",
```

Comment:
- [ ] Please include the necessary run commands in scripts:
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```

Reply:
Done, thanks for this